opennms-webapp/src/main/webapp/includes/bootstrap-footer.jsp

```diff
@@ -62,10 +63,10 @@
 
     <footer id="footer">
       <p>
-        OpenNMS <a href="support/about.jsp">Copyright</a> &copy; 2002-2017
+        OpenNMS <a href="about/index.jsp">Copyright</a> &copy; 2002-2017
         <a href="http://www.opennms.com/">The OpenNMS Group, Inc.</a>
         OpenNMS&reg; is a registered trademark of
-        <a href="http://www.opennms.com">The OpenNMS Group, Inc.</a>
+        <a href="http://www.opennms.com">The OpenNMS Group, Inc.</a> - Version: <%=Vault.getProperty("version.display")%>
```

Comment:
Can you add a screenshot of what this looks like?

Reply:
I have added detailed screenshots in the JIRA issue. I've tried to use as much plain bootstrap components as possible.